server/src/controllers/Account.ts

```diff
@@ -18,29 +21,58 @@ import * as Auth from '../lib/auth';
 @JsonController()
 export class AccountController {
   @Get('/accounts/')
+  @OpenAPI({
+    summary: 'Return a list of accounts',
+    description: 'List all accounts',
+  })
   public async getAll() {
     return Account.find();
   }
 
   @Post('/accounts/')
+  @OpenAPI({
+    summary: 'Add one account',
+    description: 'Add an account to the Account table',
+  })
+  @ResponseSchema(Account, {
+    contentType: 'text/csv',
+    description: 'A list of created account objects',
+    statusCode: '201',
+  })
   public save(@EntityFromBody() account: Account) {
     return account.save();
   }
 
   @Get('/accounts/:id/')
+  @OpenAPI({
+    summary: 'Return a single account with given ID',
+    description: 'List the account with given account ID',
+  })
   public async get(@Param('id') id: string) {
     return Account.findOne({ id });
   }
 
   @Patch('/accounts/:id/')
+  @OpenAPI({
+    summary: 'Update an account with given ID',
+    description: 'Update an account with given ID with given account object',
+  })
   public async patch(@Param('id') id: string, @Body() account: object) {
     await Account.update(id, account);
     return Account.findOne({ id });
   }
 
   @Delete('/accounts/:id/')
   @OnUndefined(204)
+  @OpenAPI({
+    summary: 'Delete an account with given ID',
+    description: 'Delete an account with given ID',
+  })
   public async remove(@Param('id') id: string) {
     return Account.delete({ id });
   }
 }
+
+const storage = getMetadataArgsStorage();
```

Comment:
Is there anyway we can add this as a route in the `app.ts`. Something like http://github.com/epiphone/routing-controllers-openapi/blob/master/sample/01-bacsic/app.ts so it will do it for all applications automatically.